pep-0561.rst

```diff
@@ -94,7 +106,9 @@ a normal package. The normal resolution order of checking ``*.pyi`` before
 ``*.py`` will be maintained. The value of the ``typed`` argument to 
 ``setup()`` is ``"stubs"`` for this type of distribution. The author of the
 package is suggested to use ``package_data`` to assure the stub files are
-installed alongside the runtime Python code.
+installed alongside the runtime Python code. In addition, the package should
```

Comment:
I think this should go in the next section (which indicates that the title of this section is confusing, since it's actually about packages containing both code and stub files: maybe it should be "Packages with Stubs" or something).

Reply:
Good catch, that was meant for the next section!